app/policies/third_party_project_policy.rb

```diff
@@ -1,2 +1,14 @@
 class ThirdPartyProjectPolicy < ProjectPolicy
+  def create?
+    !beis_user? && editable_report?
+  end
+
+  private
+
+  def editable_report?
+    # FIXME: Remove once 'level' form_step removed
+    return false if record.is_a?(Symbol)
```

Comment:
This is probably just my confusion: in which situations can the record still be a symbol here? And how does it relate to removing the `form_step`?

Reply:
You can call Pundit two ways:

- with a symbol, e.g. `policy(:third_party_project).create?` which checks to see if the current user can perform an action on *any* third-party project activity.
- with a model instance, e.g. policy(@activity).create?` you can check if the current user can perform an action on that specific third-party project activity .

The `level` form step in the wizard calls a view helper `create_activity_level_options` which is the only other place the `create?` is called on this policy.  I thought it was better to add a guard clause and comment rather than fixing tests that will I'm going to remove in my next PR.